l10n_eu_nace/models/res_partner_nace.py

```diff
@@ -46,7 +46,7 @@ def name_get(self):
     @api.model
     def name_search(self, name, args=None, operator='ilike', limit=100):
         args = list(args or [])
-        args += ['|', ('name', operator, name), ('code', 'ilike', name)]
+        args = ['|', ('code', 'ilike', name)] + args
```

Comment:
This is wrong, sorry:

```suggestion
        args = ['|', ('name', operator, name), ('code', 'ilike', name)] + args
```